rules.py

```diff
@@ -842,37 +842,22 @@ def check(self):
             else:
                 nist_obj = tag
                 object_id = element.get('objectId')
-                external_identifiers = element.find("ExternalIdentifiers")
-                if external_identifiers is None:
-                    error_message = ("Missing <ExternalIdentifiers>"
-			" tag/block in {0} with ID {1}.".format(nist_obj, object_id))
-                    self.error_log.append(base.ErrorLogEntry(
-                            element.sourceline, error_message))
+                value = element.find("ExternalIdentifiers").find("ExternalIdentifier").find("Value")
```

Comment:
Whoops, now that I look at the code more I realize that I gave you bad advice. :)

You should still check to make sure the return values are not None, but the errors that you should return should generally be of the form "${objectId} is missing a stable ExternalIdentifier".
```
ext_ids = element.Find("ExternalIdentifiers")
if not ext_ids:
  // logError("${objectId} is missing a stable ExternalIdentifier")
  continue
ext_id = ext_ids.Find("ExternalIdentifier")
if not ext_id:
  // Don't log that the ExternalIdentifier block is missing, but do:
  // logError("${objectId} is missing a stable ExternalIdentifier")
  continue
value = ext_id.Find("Value")
  // Don't log that the Value block is missing, but do:
  // logError("${objectId} is missing a stable ExternalIdentifier")
```